hal/src/main/native/athena/HAL.cpp

```diff
@@ -347,7 +347,7 @@ HAL_Bool HAL_Initialize(int32_t timeout, int32_t mode) {
 
   FRC_NetworkCommunication_Reserve(nullptr);
 
-  std::atexit([]() {
+  std::atexit([](void) {
```

Comment:
wpiformat shouldn't be doing this, and the relevant PR has been fixed. See wpilibsuite/styleguide#106.